environment.yml

```diff
@@ -7,3 +7,6 @@ dependencies:
   - pytorch=1.6.0
   - torchvision=0.7.0=py37_cpu
   - tqdm=4.47.0
+  - pip
```

Comment:
Make sure to add any new dependencies you install to `environment.yml`.  This shows how to add things you install via pip.